src/main.c

```diff
@@ -44,12 +44,15 @@ static int hours = 0;
 static int minutes = 0;
 static int day = 0;
 
+static int second_tz_offset = 0;
```

Comment:
Please can you remove this variable and use enamel_get_second_tz_offset() everywhere instead